2022-07-19 00:44:18,231 INFO    MainThread:1377950 [internal.py:wandb_internal():90] W&B internal server running at pid: 1377950, started at: 2022-07-19 00:44:18.231191
2022-07-19 00:44:18,233 INFO    WriterThread:1377950 [datastore.py:open_for_write():75] open: /home/dami/Inverse_GNN/wandb/run-20220719_004417-1kh3u4hc/run-1kh3u4hc.wandb
2022-07-19 00:44:18,234 DEBUG   SenderThread:1377950 [sender.py:send():236] send: header
2022-07-19 00:44:18,234 DEBUG   SenderThread:1377950 [sender.py:send():236] send: run
2022-07-19 00:44:18,403 DEBUG   HandlerThread:1377950 [handler.py:handle_request():140] handle_request: check_version
2022-07-19 00:44:18,408 INFO    SenderThread:1377950 [dir_watcher.py:__init__():216] watching files in: /home/dami/Inverse_GNN/wandb/run-20220719_004417-1kh3u4hc/files
2022-07-19 00:44:18,408 INFO    SenderThread:1377950 [sender.py:_start_run_threads():815] run started: 1kh3u4hc with start time 1658216657
2022-07-19 00:44:18,408 DEBUG   SenderThread:1377950 [sender.py:send():236] send: summary
2022-07-19 00:44:18,409 INFO    SenderThread:1377950 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2022-07-19 00:44:18,409 DEBUG   SenderThread:1377950 [sender.py:send_request():250] send_request: check_version
2022-07-19 00:44:18,434 DEBUG   HandlerThread:1377950 [handler.py:handle_request():140] handle_request: run_start
2022-07-19 00:44:18,435 DEBUG   HandlerThread:1377950 [meta.py:__init__():35] meta init
2022-07-19 00:44:18,435 DEBUG   HandlerThread:1377950 [meta.py:__init__():49] meta init done
2022-07-19 00:44:18,436 DEBUG   HandlerThread:1377950 [meta.py:probe():209] probe
2022-07-19 00:44:18,442 DEBUG   HandlerThread:1377950 [meta.py:_setup_git():199] setup git
2022-07-19 00:44:18,445 DEBUG   HandlerThread:1377950 [meta.py:_setup_git():206] setup git done
2022-07-19 00:44:18,446 DEBUG   HandlerThread:1377950 [meta.py:_save_pip():53] save pip
2022-07-19 00:44:18,446 DEBUG   HandlerThread:1377950 [meta.py:_save_pip():67] save pip done
2022-07-19 00:44:18,446 DEBUG   HandlerThread:1377950 [meta.py:_save_conda():74] save conda
2022-07-19 00:44:18,447 ERROR   HandlerThread:1377950 [meta.py:_save_conda():83] Error saving conda packages
Traceback (most recent call last):
  File "/home/dami/anaconda3/envs/fenics_d/lib/python3.10/site-packages/wandb/sdk/internal/meta.py", line 79, in _save_conda
    subprocess.call(
  File "/home/dami/anaconda3/envs/fenics_d/lib/python3.10/subprocess.py", line 345, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/home/dami/anaconda3/envs/fenics_d/lib/python3.10/subprocess.py", line 966, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/home/dami/anaconda3/envs/fenics_d/lib/python3.10/subprocess.py", line 1842, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'conda'
2022-07-19 00:44:18,447 DEBUG   HandlerThread:1377950 [meta.py:_save_conda():84] save conda done
2022-07-19 00:44:18,447 DEBUG   HandlerThread:1377950 [meta.py:probe():247] probe done
2022-07-19 00:44:18,449 DEBUG   SenderThread:1377950 [sender.py:send():236] send: files
2022-07-19 00:44:18,449 INFO    SenderThread:1377950 [sender.py:_save_file():952] saving file wandb-metadata.json with policy now
2022-07-19 00:44:18,453 DEBUG   HandlerThread:1377950 [handler.py:handle_request():140] handle_request: stop_status
2022-07-19 00:44:18,453 DEBUG   SenderThread:1377950 [sender.py:send_request():250] send_request: stop_status
2022-07-19 00:44:18,573 DEBUG   SenderThread:1377950 [sender.py:send():236] send: telemetry
2022-07-19 00:44:18,897 INFO    Thread-11 :1377950 [upload_job.py:push():138] Uploaded file /tmp/tmp65tdmrfzwandb/3j1mtfsl-wandb-metadata.json
2022-07-19 00:44:19,410 INFO    Thread-7  :1377950 [dir_watcher.py:_on_file_created():269] file/dir created: /home/dami/Inverse_GNN/wandb/run-20220719_004417-1kh3u4hc/files/wandb-summary.json
2022-07-19 00:44:19,410 INFO    Thread-7  :1377950 [dir_watcher.py:_on_file_created():269] file/dir created: /home/dami/Inverse_GNN/wandb/run-20220719_004417-1kh3u4hc/files/requirements.txt
2022-07-19 00:44:19,410 INFO    Thread-7  :1377950 [dir_watcher.py:_on_file_created():269] file/dir created: /home/dami/Inverse_GNN/wandb/run-20220719_004417-1kh3u4hc/files/output.log
2022-07-19 00:44:19,410 INFO    Thread-7  :1377950 [dir_watcher.py:_on_file_created():269] file/dir created: /home/dami/Inverse_GNN/wandb/run-20220719_004417-1kh3u4hc/files/wandb-metadata.json
2022-07-19 00:44:19,410 INFO    Thread-7  :1377950 [dir_watcher.py:_on_file_created():269] file/dir created: /home/dami/Inverse_GNN/wandb/run-20220719_004417-1kh3u4hc/files/conda-environment.yaml
2022-07-19 00:44:21,409 INFO    Thread-7  :1377950 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/dami/Inverse_GNN/wandb/run-20220719_004417-1kh3u4hc/files/output.log
2022-07-19 00:44:23,410 INFO    Thread-7  :1377950 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/dami/Inverse_GNN/wandb/run-20220719_004417-1kh3u4hc/files/output.log
2022-07-19 00:44:25,410 INFO    Thread-7  :1377950 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/dami/Inverse_GNN/wandb/run-20220719_004417-1kh3u4hc/files/output.log
2022-07-19 00:44:27,411 INFO    Thread-7  :1377950 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/dami/Inverse_GNN/wandb/run-20220719_004417-1kh3u4hc/files/output.log
2022-07-19 00:44:27,484 WARNING MainThread:1377950 [internal.py:wandb_internal():153] Internal process interrupt: 1
2022-07-19 00:44:27,615 DEBUG   SenderThread:1377950 [sender.py:send():236] send: telemetry
2022-07-19 00:44:27,994 WARNING MainThread:1377950 [internal.py:wandb_internal():153] Internal process interrupt: 2
2022-07-19 00:44:27,995 ERROR   MainThread:1377950 [internal.py:wandb_internal():156] Internal process interrupted.
2022-07-19 00:44:28,616 INFO    WriterThread:1377950 [datastore.py:close():279] close: /home/dami/Inverse_GNN/wandb/run-20220719_004417-1kh3u4hc/run-1kh3u4hc.wandb
2022-07-19 00:44:28,616 INFO    SenderThread:1377950 [sender.py:finish():1114] shutting down sender
2022-07-19 00:44:28,616 INFO    SenderThread:1377950 [dir_watcher.py:finish():353] shutting down directory watcher
2022-07-19 00:44:28,616 INFO    MainThread:1377950 [internal.py:handle_exit():80] Internal process exited
